### PR TITLE
vg/vgpdf: implement multipage PDFs

### PR DESCRIPTION
**Pull request by sbinet**
Fixes gonum/plot#450.

Please take a look.

### CODE REVIEW

**Comment by sbinet** [comment]
@aphecetche PTAL

File: vg/vgpdf/vgpdf_test.go [hunk 1]
```diff
@@ -169,3 +171,73 @@ func TestArc(t *testing.T) {
 		t.Fatalf("plot mismatch")
 	}
 }
+
+// Example_multipage shows how one can create a PDF with multiple pages.
+func Example_multipage() {
+	c := vgpdf.New(5*vg.Centimeter, 5*vg.Centimeter)
+
+	{
```
Comment (kortschak) [inline vg/vgpdf/vgpdf_test.go]: Can you do this with a `i, c := range []color.RGBA{{B: 255, A: 255}, {R: 255, A: 255}}`? It would be cleaner and clearer.

Comment (sbinet) [reply vg/vgpdf/vgpdf_test.go]: done.

**Comment by sbinet** [comment]
PTAL

File: vg/vgpdf/vgpdf_test.go [hunk 2]
```diff
@@ -169,3 +171,54 @@ func TestArc(t *testing.T) {
...
 	}
 }
+
+// Example_multipage shows how one can create a PDF with multiple pages.
+func Example_multipage() {
+	c := vgpdf.New(5*vg.Centimeter, 5*vg.Centimeter)
+
+	for i, col := range []color.RGBA{{B: 255, A: 255}, {R: 255, A: 255}} {
+		if i > 0 {
+			// add a new page.
```
Comment (kortschak) [inline vg/vgpdf/vgpdf_test.go]: Initial capital.

File: vg/vgpdf/vgpdf_test.go [hunk 3]
```diff
@@ -169,3 +171,54 @@ func TestArc(t *testing.T) {
...
+		if err != nil {
+			log.Fatalf("could not create line: %v", err)
+		}
+		line.Color = col
+		p.Add(line)
+		p.Title.Text = fmt.Sprintf("Plot %d", i+1)
+		p.X.Label.Text = "X axis"
+		p.Y.Label.Text = "Y axis"
+
+		// write plot to page
```
Comment (kortschak) [inline vg/vgpdf/vgpdf_test.go]: Initial capital and full stop.

**Comment by sbinet** [comment]
PTAL

**Pull request merged**